charmhelpers/contrib/network/ip.py

```diff
@@ -469,9 +469,18 @@ def ns_query(address):
         return None
 
     try:
-        answers = dns.resolver.query(address, rtype)
+        resolv = dns.resolver.Resolver()
```

Comment:
While im confident that 5 seconds is the default dns timeout in Ubuntu I think this needs a comment to say that and if possible how this can be asserted.